src/foundry/client/pixi/extensions/circle.d.mts

```diff
@@ -111,10 +111,11 @@ declare module "pixi.js" {
      */
     intersectClipper(
       clipperPoints: PIXI.Polygon.ClipperPoint[],
-      options?: {
-        /** The number of points which defines the density of approximation */
-        density: number;
-      },
+      options?: Parameters<PIXI.Polygon["intersectClipper"]>[1] &
```

Comment:
nit, I'd make a type in the `Polygon` namespace (if one doesn't exist already) so you can write `PIXI.Polygon.IntersectClipperOptions`.